src/app/core/user/user-select/user-select.component.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  Component,
+  EventEmitter,
+  Input,
+  OnInit,
+  Output,
+  ViewChild,
+} from "@angular/core";
+import { EntityMapperService } from "../../entity/entity-mapper.service";
+import { User } from "../user";
+import { MatAutocompleteTrigger } from "@angular/material/autocomplete";
+import { AlertService } from "../../alerts/alert.service";
+import { LoggingService } from "../../logging/logging.service";
+
+@Component({
+  selector: "app-user-select",
+  templateUrl: "./user-select.component.html",
+  styleUrls: ["./user-select.component.scss"],
+})
+export class UserSelectComponent implements OnInit {
+  @Input() user: string;
+  @Output() userChange = new EventEmitter<string>();
```

Comment:
That was not clearly specified in the issue, but we have cases where more than one user is selected so we will need to support multi-select somehow

Reply:
The Note "author" is actually misnamed currently, it is rather used as a description of staff members (= users) who have been part of the real-world interaction the Note describes. Therefore more than one user can be linked.